crypto/crypto.go

```diff
@@ -139,6 +139,11 @@ func LoadECDSA(file string) (*ecdsa.PrivateKey, error) {
 	return ToECDSA(buf), nil
 }
 
+// SaveECDSA saves a secp256k1 private key from the given file.
+func SaveECDSA(file string, key *ecdsa.PrivateKey) error {
+	return common.WriteFile(file, FromECDSA(key))
```

Comment:
Looking further, `common.WriteFile` is another one of those shortcuts that could be replaced with `ioutil.WriteFile`.
For writing private keys, it's probably in the interest of the user to create the file with non world-readable permissions.

    ioutil.WriteFile(file, crypto.FromECDSA(key), 0600)